examples/chart/teleport/README.md

```diff
@@ -46,16 +46,17 @@ $ helm install --name teleport ./
 Teleport proxy generates a TLS key and a cert of its own by default.
 In order to instruct the proxy to use the TLS assets brought by you, prepare the following files:
 
-- Your CA cert named `ca.pem`
+- Your CA cert named `ca.pem`  (optional depending on values.yaml setting)
```

Comment:
I think we should be more explicit with a CA is here, and what type of CA is it.  This is Certificate Authority (CA) for SSH Certificates vs `.pems` for TLS

Reply:
I modified the writing to indicate this is the TLS CA, not SSH.

Reply:
PTAL @benarent